solidity/test/bridge/Bridge.Governance.test.ts

```diff
@@ -960,6 +960,145 @@ describe("Bridge - Governance", () => {
     )
   })
 
+  describe("beginRedemptionTxMaxTotalFeeUpdate", () => {
+    context("when the caller is not the owner", () => {
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(thirdParty)
+            .beginRedemptionTxMaxTotalFeeUpdate(1)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when the caller is the owner", () => {
+      let tx: ContractTransaction
+
+      before(async () => {
+        await createSnapshot()
+
+        tx = await bridgeGovernance
+          .connect(governance)
+          .beginRedemptionTxMaxTotalFeeUpdate(1337)
+      })
+
+      after(async () => {
+        await restoreSnapshot()
+      })
+
+      it("should not update the redemption tx max total fee", async () => {
+        const { redemptionTxMaxTotalFee } = await bridge.redemptionParameters()
+        expect(redemptionTxMaxTotalFee).to.be.equal(
+          constants.redemptionTxMaxTotalFee
+        )
+      })
+
+      it("should start the redemption tx max total fee timer", async () => {
+        const blockTimestamp = (await ethers.provider.getBlock(tx.blockNumber))
+          .timestamp
+        const initTimestamp = (await tx.wait()).events[0].args.timestamp
+        const elapsedTime = BigNumber.from(blockTimestamp).sub(initTimestamp)
```

Comment:
I don't remember exactly why I added similar tests but maybe it was a borderline "stretch" testing because we execute `RedemptionTxMaxTotalFeeUpdateStarted` event in a separate library and now we check that everything is happening in the same transaction and the delay time was not affected. `elapsedTime` should be zero if everything works as expected.

Reply:
I think we should remove it, the transaction is a part of the block and the EVM guarantees the timestamps match.

Reply:
https://github.com/keep-network/tbtc-v2/pull/378